common/browser.go

```diff
@@ -414,6 +408,7 @@ func (b *Browser) Close() {
 	// will stop emitting events.
 	b.browserProc.GracefulClose()
 	b.browserProc.Terminate()
+	b.conn.Close()
```

Comment:
It makes sense to put this here since the browser is the one that opens up a new connection.